### PR TITLE
[ANALYZER-3828] - Saving an inline model breaks localization

### PR DESCRIPTION
**Pull request by bcostahitachivantara**
@LeonardoCoelho71950 
@lucboudreau 
@pentaho-lmartins 
@pentaho/tatooine

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 3m 57s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl mondrian
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 3m 50s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl mondrian
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: mondrian/src/main/java/mondrian/rolap/RolapSchemaPool.java [hunk 1]
```diff
@@ -366,7 +366,9 @@ private static String getSchemaContent(
             }
         }
         else {
-            catalogStr = catalogStr = processDynamicSchema( dynProcName, null, catalogStr, connectInfo);
+            if (!Util.isEmpty(dynProcName)) {
+                catalogStr = catalogStr = processDynamicSchema( dynProcName, null, catalogStr, connectInfo );
```
Comment (mkambol) [inline mondrian/src/main/java/mondrian/rolap/RolapSchemaPool.java]: Why `catalogStr = catalogStr =`?

Comment (bcostahitachivantara) [reply mondrian/src/main/java/mondrian/rolap/RolapSchemaPool.java]: it was a mistake, let me fix that

**Comment by buildguy** [comment]
### :gear: Build finished in 3m 37s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl mondrian
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Pull request merged**